.babelrc-v9.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26962&commitId=d66e9eedc0dc9c315ba623de76a907d01c37dd48&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1855 screenshots unchanged, 0 screenshots with different dimensions and 2 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.size - RTL.normal.chromium.png|118| Changed|
|Accordion Converged.size.normal.chromium.png|251| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>